HarfangLab/harfanglab/asset_connector/device_assets.py

```diff
@@ -107,6 +107,8 @@ def map_fields(self, asset: dict[str, Any]) -> DeviceOCSFModel:
         return:
             DeviceOCSFModel: The mapped OCSF Device model.
         """
+        self.log(f"Mapping fields for asset: {asset.get('hostname', 'Unknown Hostname')}")
```

Comment:
Can you provide the level for all log calls?
```suggestion
        self.log(f"Mapping fields for asset: {asset.get('hostname', 'Unknown Hostname')}", level="debug")
```